src/libcore/char_private.rs

```diff
@@ -8,8 +8,8 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-// NOTE: The following code was generated by "src/etc/char_private.py",
-//       do not edit directly!
+//! NOTE: The following code was generated by `src/etc/char_private.py`;
+//!       do not edit directly!
```

Comment:
This I believe wasn't intended to be a doc comment, can the `!` be removed.

Reply:
Okay!